VDA5050_EN.md

```diff
@@ -903,7 +933,21 @@ An exception to this rule is, if the AGV has to pause on the edge (because of a
 ![Figure 13 nodeStates, edgeStates, actionStates during order handling](./assets/Figure13.png)
 >Figure 13 nodeStates, edgeStates, actionStates during order handling
 
+#### Corridors
+
+The corridor attribute of an edge leads to two different types of nodes inside inside an order: *goal nodes* and *way nodes*.
+
+- A *goal node* contains actions and therefore the shuttle has to reach this nodes precisely. The fleet management can define via the `allowedDeviationXY` and `allowedDeviationTheta` how precisely the node has to be reached. If no deviation is defined no deviation is allowed (no deviation means within the normal tolerance of the AGV manufacturer).
+- A *way node* contains no actions and therefore the shuttle may pass this node not precisely. The node attributes  `allowedDeviationXY` and `allowedDeviationTheta`  have no effect.
+*(Remarks: Consulting  `allowedDeviationXY` and `allowedDeviationTheta` to detect whether a way point is passed or not makes it difficult to define a correct ridable order and not defeating the reason using the corridor attribute. While the corridor attribute covers the whole vehicle contour an allowed deviation refers to the reference point of the vehicle. This makes it not trivial to define right deviation range according to the given corridor polygon or vice versa. Way points should be used defining the navigation area together with the corridor attribute and traffic control.)*
+
+An AGV on an edge with a corridor attribute shall determine the current edge by calculating the perpendicular distance to each edge of the current base. The edge with the shortest distance is the current edge. All nodes and edges with lower sequence numbers count as passed.
+
+An AGV coming from an edge without a corridor attribute is not allowed to use the corridor of a subsequent edge until the current edge is released.
```

Comment:
An AGV/AMR should not travel on a release edge. The polygon of the new edge must cover the vehicle position at this point.
The situation for an AGV with the current standard is quite similar. If the AGV is inside the deviation range of a node the node is reached an the incoming edge is release. The AGV might still standing physically on the old edge but is already reporting the next edge. (I think using the perpendicular distance leads to an later switch to the next edge than using the 'devationRangeXy' in case of an AGV.)

Reply:
```
An AGV coming from an edge without a corridor attribute is not allowed to use the corridor of a subsequent edge until the current edge is released.
```
This sentence requires that the current edge is released. The current edge is the edge the vehicle is on. In this description the current edge would be the edge before the corridor. Or am i mistaken?
Given this interpretation the proposed sentence does not make sense.

Reply:
Maybe I should refer to the node (like the current standard text), than it is clear.